tsqa/test_cases.py

```diff
@@ -46,7 +46,7 @@ class EnvironmentCase(unittest.TestCase):
     def run(self, result=None):
         unittest.TestCase.run(self, result)
         # we want to keep track of failures at a class level-- not instance level
-        self.__class__.__successful &= result.result.wasSuccessful()
+        self.__class__.__successful &= result.wasSuccessful()
```

Comment:
Oh, sorry, I did not give the right link. I'm using python 2.6.6
https://docs.python.org/2.6/library/unittest.html?highlight=unittest#unittest.TestResult.wasSuccessful